sql/mysql_db/privilege_set.go

```diff
@@ -134,7 +136,7 @@ func (ps PrivilegeSet) RemoveGlobalDynamic(privileges ...string) {
 // RemoveDatabase removes the given database privilege(s).
 func (ps PrivilegeSet) RemoveDatabase(dbName string, privileges ...sql.PrivilegeType) {
 	// We don't use the getUseableDb function since we don't want to create a new map if it doesn't already exist
-	dbSet := ps.Database(dbName)
+	dbSet := ps.Database(dbName).(PrivilegeSetDatabase)
```

Comment:
These type assertions on an unchecked DB name in a public method look panicky to me